tests/Feature/Traits/CatchFailures.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Feature\Traits;
+
+use Illuminate\Testing\TestResponse;
+
+trait CatchFailures
+{
+	protected function assertStatus(TestResponse $response, int $expectedStatusCode): void
+	{
+		if ($response->getStatusCode() === 500) {
+			$exception = $response->json();
+			$this->trimException($exception);
+			// $exception['trace'] = array_slice($exception['trace'], 0 ,3);
+			dump($exception);
```

Comment:
This very much looks suspiciously like the ill-reputed "dump & die" debugging approach of ancient PHP. While I don't care whether anybody does so in private during development, I would not like to see that in our "official" code base even if it is only in test code.

Reply:
This is not dump & die (as we don't die).
It just dump the exception in the case of a 500 when running the tests.
This makes it way it easier to figure what went wrong rather than the E fro phpunit.